### PR TITLE
35 launch screen design

### LINKED ISSUE #35: Add an opening page when opening the app
After the logo is completed, we can start adding an opening page as a transition to the main view pages.

Read over here: https://developer.apple.com/documentation/xcode/specifying-your-apps-launch-screen

Essentially, what we need to do: 
1. Go to Info > Custom iOS Target Properties
2. Check Launch Screen (should be a dictionary), then add "Image Name" with the actual name of the image
3. We cannot simply add "AppIcon" to the launch screen, because it would be too big, we would need a folder containing different size of the same image.

Here is a great video tutorial: https://www.youtube.com/watch?v=1zqxW1ADRzk&t=378s 

- [x] Wait to do this issue until this is done: https://github.com/algebra2boy/ShopHub/issues/18

### PR DESCRIPTION
**Pull request by TaoLyn838**
https://github.com/algebra2boy/ShopHub/assets/58400041/7669f7da-41f3-45a6-ae4d-ab43951e12c7

### CODE REVIEW

File: ShopHub/ShopHub/Info.plist [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>UILaunchScreen</key>
```
Comment (algebra2boy) [inline ShopHub/ShopHub/Info.plist:5]: Great addition to the Launch Screen!

File: ShopHub/ShopHub/LaunchScreen/LaunchScreenView.swift [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+                    secondPhaseIsAnimating.toggle()
+                }
+            default: break
+            }
+        }
+        .opacity(secondPhaseIsAnimating ? 0 : 1)
+    }
+}
+
+private extension LaunchScreenView {
```
Comment (algebra2boy) [inline ShopHub/ShopHub/LaunchScreen/LaunchScreenView.swift:40]: great encapsulation.

**Review by algebra2boy — APPROVED** [review]
LGTM!

File: ShopHub/ShopHub/ContentView.swift [hunk 3]
```diff
@@ -15,18 +15,16 @@ struct ContentView: View {
...
             MenuTabView()
-                .onAppear {
-                    DispatchQueue
-                        .main
-                        .asyncAfter(deadline: .now() + 3) {
-                            launchScreenManager.dismiss()
-                        }
-                }
         }
+        .onAppear(perform: dimissLaunchScreen)
```
Comment (TaoLyn838) [inline ShopHub/ShopHub/ContentView.swift:19]: Wow, I didn't realize these parameters can do the same jobs. Now, I learned them, Thanks!

**Review by TaoLyn838 — COMMENTED** [review]
Great optimizations!!!

**Pull request merged**